Koans/AboutClasses.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using DotNetCoreKoans.Engine;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+    public class AboutClasses : Koan
+    {
+
+        class Foo1
+        {
+
+        }
+
+        /// <summary>
+        /// Objects instantiation.
+        /// </summary>
+        [Step(1)]
+        public void InstancesOfAClassesCanBeCreatedWithNew()
+        {
+            object foo = null;
+            Assert.NotNull(foo);
+        }
+
+        class Foo2
+        {
+            public int Int { get; set; }
+            internal string _str;
+        }
+
+        /// <summary>
+        /// Instance members.
+        /// </summary>
+        [Step(2)]
+        public void InstanceMembersCanBeSetByAssigningToThem()
+        {
+            var foo = new Foo2();
+            Assert.Equal(1, foo.Int);
+            Assert.Equal("Bar", foo._str);
+        }
+
+        class Foo3
+        {
+            private bool _boom = true;
+            public bool Internal { get => _boom; set { _boom = value; } }
+
+            public void Do()
+            {
+                if (_boom)
+                {
+                    throw new InvalidOperationException(nameof(Do));
+                }
+            }
+        }
+
+        /// <summary>
+        /// Accessors.
+        /// </summary>
+        [Step(3)]
+        public void UseAccessorsToReturnInstanceVariables()
+        {
+            var foo = new Foo3();
+            // make sure it won't explode
+            foo.Do();
```

Comment:
Might be better to do an exception assert here.

Reply:
The goal is to write a test that doesn't throw an exception.
There is no `Assert.NotThrows` in xunit.
`Assert.Throws(typeof(InvalidOperationException), () => foo.Do());`